addons/website/static/src/xml/website.editor.xml

```diff
@@ -108,4 +111,13 @@
             </div>
         </div>
     </div>
+    <!-- Add a Google Font option dialog -->
+    <div t-name="website.dialog.addGoogleFont">
+        <div class="form-group row">
+            <div class="col-md-12">
```

Comment:
row col-md-12 ?

Reply:
If I do that `text-right` on span below won't work anymore, right ? Maybe we don't care to have it on the right anyway :smile: ?

Reply:
If you do what ?

Reply:
Nothing. Sorry I did not understand at first ;)